extensions/binding/org.eclipse.smarthome.binding.digitalstrom/README.md

```diff
@@ -15,71 +15,67 @@ The digitalSTROM-Server is required for accessing any other digitalSTROM-Devices
+The digitalSTROM-Server is discovered by mDNS or *dss.local.* at the local network. Once the server is added as a thing, you have to set a user name and password or insert a valid application-token to authenticate with the server. If the binding is authorized, it automatically reads all supported devices, the dS-Meters and temperature controlled configured zones, that are set up on the digitalSTROM-System, and puts them into the *inbox*.
  
 digitalSTROM-Scenes can be discovered, too. The background scene-discovery is deactivated by default to not flood the inbox. Otherwise it will discover so many scenes, that it can be difficult to find the searched devices.
  
-Discoverable scenes are all user named scenes, group scenes that are reachable by local push-buttons, zone scenes and apartment scenes. The discovery also will discover all called scenes, if they aren't automatically discovered yet.
-
-If you only want to discover one of the thing types, you can start a discovery scan on the thing type you wish have discovered. You can use the command line command, e.g.: ``smarthome:discovery start digitalstrom:namedScene`` to start the scan. To find out what thing types this binding supports please have a look at the **Supported Things** section.
-
-## digitalSTROM-Scenes
-
-### General
-
-The device scene configuration will be saved persistently to the things to update the device state faster. For that each scene configuration of each device has to read out first, because of the digitalSTROM-rule 9 that requires a waiting period of one minute, that take some time so that at the first start a scene call can take some time to read it out and update the device state. To read it out faster only the discovered or called scenes will be read out. 
- 
-**Note:**
-Because the digitalSTROM-Server can't inform the binding about save scene events at this time, the persistently saved scene configurations can't be updated. The current troubleshooting to read out the new scene configuration after a save scene action at the digitalSTROM-Server is the following:
+Discoverable scenes are all user named scenes, group scenes that are reachable by local push-buttons, zone scenes and apartment scenes. The discovery also will discover all called scenes, if they aren't automatically discovered yet. Temperature control scenes, like *eco* will be ignored, so they can not be discoverd. 
 
-1. delete the thing to delete the persistently saved scene configuration
-2. restart the server to delete the temporary saved scene configuration 
-3. add the thing again to read out the scene configuration again.   
+If you only want to discover one of the thing types, you can start a discovery scan on the thing type you wish have discovered. You can use the command line command, e.g.: ``smarthome:discovery start digitalstrom:namedScene`` to start the scan. What thing types this binding supports please have a look at **Supported Things**. 
 
```

Comment:
... on the thing type you want to have discovered.

For thing types this binding .....